app/src/displays/datetime/datetime.vue

```diff
@@ -2,88 +2,93 @@
 	<span class="datetime">{{ displayValue }}</span>
 </template>
 
-<script lang="ts">
+<script lang="ts" setup>
 import { useI18n } from 'vue-i18n';
-import { defineComponent, ref, watch, PropType, computed } from 'vue';
+import { ref, watch, computed, onMounted, onUnmounted } from 'vue';
 import localizedFormat from '@/utils/localized-format';
 import localizedFormatDistance from '@/utils/localized-format-distance';
 import { parseISO, parse } from 'date-fns';
 
-export default defineComponent({
-	props: {
-		value: {
-			type: String,
-			required: true,
-		},
-		type: {
-			type: String as PropType<'dateTime' | 'time' | 'date' | 'timestamp'>,
-			required: true,
-			validator: (val: string) => ['dateTime', 'date', 'time', 'timestamp'].includes(val),
-		},
-		format: {
-			type: String,
-			default: 'long',
-		},
-		relative: {
-			type: Boolean,
-			default: false,
-		},
-	},
-	setup(props) {
-		const { t } = useI18n();
-
-		const localValue = computed(() => {
-			if (!props.value) return null;
-
-			if (props.type === 'timestamp') {
-				return parseISO(props.value);
-			} else if (props.type === 'dateTime') {
-				return parse(props.value, "yyyy-MM-dd'T'HH:mm:ss", new Date());
-			} else if (props.type === 'date') {
-				return parse(props.value, 'yyyy-MM-dd', new Date());
-			} else if (props.type === 'time') {
-				return parse(props.value, 'HH:mm:ss', new Date());
+interface Props {
+	value: string;
+	type: 'dateTime' | 'date' | 'time' | 'timestamp';
+	format?: string;
+	relative?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+	format: 'long',
+	relative: true,
```

Comment:
Changing the default of the `relative` prop to `true` causes the datetime display to always be relative because the prop default and the display option default aren't the same. When the display option for _relative_ is turned of, it will simply use the component's default value (which is `true`) and when it is turned on, it will also be `true`.

Reply:
Ah nice catch. Thought it was a 1-to-1 change, but it's definitely always true now.. Removing it should be the way to go right? Tested that and it seems to work.

Reply:
Ref 1486d37eb

Reply:
scratch that... for some reason the default is `true` in my head, even when it was `false` all along 🤦‍♂️